src/WebWindow/WebWindow.cs

```diff
@@ -56,6 +56,9 @@ public class WebWindow
         [UnmanagedFunctionPointer(CallingConvention.Cdecl)] delegate void ResizedCallback(int width, int height);
         [UnmanagedFunctionPointer(CallingConvention.Cdecl)] delegate void MovedCallback(int x, int y);
 
+        [UnmanagedFunctionPointer(CallingConvention.Cdecl, CharSet = CharSet.Auto)]
+        delegate void UriChangeCallback(string uri); 
```

Comment:
Could you follow the same layout as on the other lines? (i.e., single line for this, for consistency